tests/e2e/Services/ProjectWebhooks/WebhooksCustomServerTest.php

```
@@ -0,0 +1,952 @@
+        $index = $this->client->call(Client::METHOD_POST, '/databases/' . $databaseId . '/collections/' . $actorsId . '/indexes', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'key' => 'fullname',
+            'type' => 'key',
+            'attributes' => ['lastName', 'firstName'],
+            'orders' => ['ASC', 'ASC'],
+        ]);
+
+        $indexKey = $index['body']['key'];
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$indexKey'. (undefined)

(UnusedLocalVariable)